src/pages/MainPage/MainPage.tsx

```diff
@@ -128,6 +152,16 @@ export const MainPage: React.FC = () => {
             {boxes.map((box, index) => (
               <Box key={`${box}-${index}`} number={box} index={index} />
             ))}
+            <div
+              className={`main-page__play__boxes__grid__prisoner ${
+                gameRunning
+                  ? "main-page__play__boxes__grid__prisoner--active"
+                  : ""
+              }`}
+              ref={floatingPrisonerRef}
+            >
+              {currentPrisoner + 1}
```

Comment:
One Last optional feedback, maybe we should make it `P${currentPrisoner + 1}`, for distinction that this represents Prisoner

Reply:
Similarly if you want we can add `B` as prefix on Boxes

Reply:
Arigato! Will add this in next PR